File: Difficulty: Hard/Median in a row-wise sorted Matrix/median-in-a-rowwise-sorted-matrix.java
import java.util.Arrays;

class Solution {
    int median(int mat[][]) {
      
        int rows = mat.length;
        int cols = mat[0].length;
        int arrLength = rows*cols;
    
        int[] arr = new int[arrLength];

        int index = 0;
        for (int[] row : mat) {
            for (int num : row) {
                arr[index++] = num;
            }
        }

        Arrays.sort(arr);
        
        int n = arr.length;
        return arr[n / 2];
      
    }
}

File: Difficulty: Hard/Median in a row-wise sorted Matrix/median-in-a-rowwise-sorted-matrix.py
#User function Template for python3

class Solution:
    def median(self, mat):
    	
    	arr = []
    	for i in mat:
    	    arr.extend(i)
    	    
    	arr.sort()
    	
    	n = len(arr)
    	median = arr[n//2]
    	
    	return median